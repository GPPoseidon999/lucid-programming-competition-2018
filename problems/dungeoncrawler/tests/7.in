13
a b
a a 2 c d
b b 3 e f g
c c 2 h i
d d 1 j
e e 1 j
f f 0
g g 0
h h 0
i i 0
j j 3 k l m
k k 1 c
l l 0
m m 0