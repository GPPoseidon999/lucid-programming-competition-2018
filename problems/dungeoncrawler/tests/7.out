h i c k l m j d a e f g b